sdk/tests/test_file_streaming.py

```diff
@@ -0,0 +1,139 @@
+import json
+import logging
+import os
+import unittest
+
+from lusid_drive import ApiConfigurationLoader, SearchApi, SearchBody, FilesApi, FoldersApi, ApiException, models
+from lusid_drive.utilities import ApiClientFactory, get_file_id, get_folder_id
+from lusid_drive.utilities.file_streaming import stream_file_upload
+
+
+class FileStreaming(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.logger = logging.getLogger()
+        cls.logger.setLevel(logging.INFO)
+        config = ApiConfigurationLoader.load("C:\work\code\drive-sdk-python-preview\secrets.json")
```

Comment:
use `path.join()` to make it platform agnostic and also relative to the current folder